tests/features/steps/basic_commands.py

```diff
@@ -60,3 +61,5 @@ def step_send_help(context):
     Send \? to see help.
     """
     context.cli.sendline('\\?')
+    wrappers.expect_exact(
```

Comment:
You're using the same "expect something bounded by a boundary" pattern so many times, it may deserve a separate wrapper, like `wrappers.expect_exact_with_boundary`.